src/components/global/SandboxPage/SandboxPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { SandboxIframe } from "../../utility/SandboxIframe/SandboxIframe"; 
+
+interface SandboxPageProps {
+    title: string,
+    description: {
+        before: React.ReactNode;
+        after: React.ReactNode;
+    },
+    sandbox: string
+}
+
+export default function SandboxPage({title, description, sandbox}:SandboxPageProps) {
```

Comment:
@hubert-deriv 
```suggestion
export default function SandboxPage({title, description, sandbox}: SandboxPageProps) {
```
nit🙂

Reply:
fixed